nomenclature/processor/region.py

```diff
@@ -93,7 +93,7 @@ class RegionAggregationMapping(BaseModel):
         Optionally, list of common regions where aggregation will be performed.
     """
 
-    model: str
+    model: Union[List[str], str]
```

Comment:
Just one (mostly stylistic) comment. Instead of allowing both `str` and `List[str]` you can also just allow a list of strings:

```suggestion
    model: List[str]
```

in combination with a pre validator that converts it to a list if it is provided as just a string

```python

@validator("model", pre=True)
def convert_to_list(cls, v):
    return pyam.utils.to_list(v)
```

this would mean that for any interaction with `RegionAggregationMapping` you can be sure that `model` is a list and iterate over it directly instead of having to check if it is a string or a list.

Reply:
Thanks for the suggestion - implemented.